src/ripple/basics/impl/BasicConfig.cpp

```diff
@@ -134,6 +116,17 @@ BasicConfig::overwrite (std::string const& section, std::string const& key,
 }
 
 void
+BasicConfig::legacy(std::string const& section, std::string value)
+{
+    map_[section].legacy(std::move(value));
+}
+
+std::string BasicConfig::legacy(std::string const& sectionName) const
```

Comment:
the reason is that `void` or `auto` only convey a tiny bit of information, there's nothing to "go wrong", and there are only a limited number of lines on the screen.

It's funny you believe this and at the same time want to jam as many function arguments into one line of a declaration as possible, even though those are long, repetitive and prone to error!